ezflow/data/dataset/base_dataset.py

```diff
@@ -101,6 +106,17 @@ def __getitem__(self, index):
         img1 = np.array(img1).astype(np.uint8)
         img2 = np.array(img2).astype(np.uint8)
 
+        img1 = torch.from_numpy(img1).permute(2, 0, 1).float()
```

Comment:
let's reduce the duplicate operations of converting to and from numpy to pytorch. We can move the code to make grayscale image to rgb at `line:120` before `line 109`

Reply:
Done